src/components/track-tile/TrackTileActionButtons.tsx

```diff
@@ -173,44 +164,46 @@ export const TrackTileBottomButtons = ({
   )
 
   const shareButton = (
-    <Pressable
-      disabled={disabled}
-      onPress={() => {
-        if (onPressShare) {
-          toast({
-            content: messages.copiedToast,
-            timeout: SHARE_TOAST_TIMEOUT
-          })
-          onPressShare()
-        }
-      }}
-    >
-      <IconShare
-        height={18}
-        width={18}
-        fill={neutralLight4}
-        style={[styles.button, disabled ? { opacity: 0.5 } : {}]}
-      />
-    </Pressable>
+    <IconButton
+      isDisabled={disabled}
+      onPress={onPressShare}
+      icon={() => (
+        <IconShare
+          height={18}
+          width={18}
+          fill={neutralLight4}
+          style={[styles.button, disabled ? { opacity: 0.5 } : {}]}
+        />
+      )}
+    />
   )
 
   const moreButton = (
-    <Pressable onPress={onPressOverflow} disabled={disabled}>
-      <IconKebabHorizontal
-        height={22}
-        width={22}
-        fill={neutralLight4}
-        style={[
-          styles.button,
-          styles.lastButton,
-          disabled ? { opacity: 0.5 } : {}
-        ]}
-      />
-    </Pressable>
+    <IconButton
+      onPress={onPressOverflow}
+      isDisabled={disabled}
+      icon={() => (
+        <IconKebabHorizontal
+          height={22}
+          width={22}
+          fill={neutralLight4}
+          style={[
+            styles.button,
+            styles.lastButton,
+            disabled ? { opacity: 0.5 } : {}
```

Comment:
does it make sense to add this styling option to the iconbutton when disabled? i guess its not really possible to add that style though huh, maybe `cloneElement`, anyway...

Reply:
I agree, the only reason I'm using a render function here is because IconButton isn't very flexible and doesn't accept a fill. I can address that in another pr

Reply:
awesome sounds like a plan